recipes/mummer/build.sh

```diff
@@ -48,6 +48,12 @@ for i in exact-tandems dnadiff mapview mummerplot nucmer promer run-mummer1 run-
 done
 
 for i in $binaries; do 
+  if file $MUMMER_HOME/$i | grep "Perl script"; then
```

Comment:
Since `binaries` are manually listed above, would it make sense to have a separate list for binaries and perl scripts so it's explicit which is which?

Reply:
It crossed my mind too, and a hard coded list is ok given mummer3 is no longer under active development